adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -39,9 +39,54 @@ public void InitializeBeforeTest()
         /// Runs through the standard acquire token flow
         /// </summary>
         [Test]
-        public void AcquireTokenTest()
+        public void AcquireTokenInteractiveTestADAL()
```

Comment:
Any reason you are not calling the tests the same across libraries? Wouldn't that make things simpler when moving towards the unified app?